fidesctl/tests/core/test_apply.py

```diff
@@ -112,3 +189,27 @@ def test_execute_create_update_unchanged_empty():
         url="test", headers={"test": "test"}, resource_type="test"
     )
     assert True
+
+
+@pytest.mark.unit
+def test_returns_nested_fields(test_unnested_fields, test_nested_fields):
```

Comment:
This was really the first area I was looking at as part of my initial testing (before moving to the `evaluate` command), without the change to the model the `apply` command would succeed but no nested fields were captured and sent to the database. 

I can see this test not being needed or maybe misplaced (more to do with the model than the apply command) but wanted to add some protection in case of future model changes - is there a better place to do that or should this just be removed you think?

Reply:
I would lean towards removing it since there's nothing about this model change that I'm thinking needs special consideration

Reply:
I'd leave this up to you though

Reply:
Cool, thank you! Will remove it now 👍🏽